.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a question
+about: Asking a question related gRPC-java
+labels: question
+---
+
+Please answer these questions before submitting a bug report.
+
+For non direct gRPC-java related questions, please use [stackoverflow](stackoverflow.com).
```

Comment:
"For questions not directly related to gRPC-Java,..."

Reply:
done.